wnfs/src/lib.rs

```diff
@@ -26,20 +26,10 @@ pub use traits::*;
 // Re-exports
 //--------------------------------------------------------------------------------------------------
 
-pub mod ipld {
-    pub use libipld::{
-        cbor::DagCborCodec,
-        cid::Version,
-        codec::{Codec, Decode, Encode},
-        serde::Serializer,
-        Cid, Ipld, IpldCodec,
-    };
-}
+pub use libipld;
 
-pub mod rng {
+pub mod rand_core {
     pub use rand_core::RngCore;
 }
 
-pub mod ratchet {
-    pub use skip_ratchet::{Ratchet, RatchetSeeker};
-}
+pub use skip_ratchet;
```

Comment:
I've been thinking about these re-exports lately when I looked at the public API:
I'm on board with the `libipld` and `rand_core` exports, since they appear in `wnfs`'s public api. However, ideally a downstream user of `wnfs` wouldn't need to worry about knowing what a `skip_ratchet` is or does, so perhaps we shouldn't expose it?